### ISSUE TITLE
stakepools: note how new pools are listed

### ISSUE THREAD

**Issue by xaur** [decred/dcrweb#414]
People often ask how to get their stake pools listed. Adding short (1-2 sentences with a link) note to the [stakepools](https://decred.org/stakepools/) page would inform them earlier and save a bit of time for all.

**Comment by xaur** [comment]
@s-ben I was referring to decred.org/stakepools . I believe the source is [here](https://github.com/decred/dcrweb/blob/master/src/stakepools/index.html).

**Comment by ZeroASIC** [comment]
I think it should go below the list on decred.org/vsp/ but I need two sentences to add.  How do they get added?  Do they just find us on slack or matrix?  It should contain the appropriate link.  I can add it as soon as we decide what the copy should say.  Something like: "To get your site added to this list just find us on matrix. [link]"

**Comment by xaur** [comment]
How about

> To get your VSP listed, you need at least 3 geographically distributed voting wallets and at least 1 ticket that has voted on mainnet. After that, please join #pos-voting room and ask @dhill: [Matrix link](https://matrix.to/#/!wSdymYrEpBhsWlDJuk:decred.org), [Slack link](https://decred.slack.com/messages/C3ZNV8QTX/).

```md
To get your VSP listed, you need at least 3 geographically distributed voting wallets and at least 1 ticket that has voted on mainnet. After that, please join #pos-voting room and ask @dhill: [Matrix link](https://matrix.to/#/!wSdymYrEpBhsWlDJuk:decred.org), [Slack link](https://decred.slack.com/messages/C3ZNV8QTX/).
```

Correct @dajohi?

**Comment by xaur** [comment]
Oops sorry for wrong mention.

**Comment by s-ben** [comment]
Slight word tweaking on copy:

To get your VSP listed, you need to have at least three geographically-distributed voting wallets, and  at least one ticket that has successfully voted on mainnet. Once you have met these requirements, join the #pos-voting room on [Matrix](https://matrix.to/#/!wSdymYrEpBhsWlDJuk:decred.org) or [Slack](https://decred.slack.com/messages/C3ZNV8QTX/) and ask @dhill to add you.

**Comment by xaur** [comment]
Nice improvement @s-ben. Btw let's avoid mentioning wrong people by escaping their mentions: `@user`.

**Comment by ZeroASIC** [comment]
I'll update this after pull request #435 gets finished up.

**Comment by ZeroASIC** [comment]
PR #514 adds the suggested paragraph.

**Comment by ZeroASIC** [comment]
Closed in #514
